src/WebJobs.Extensions.CosmosDB/Trigger/CosmosDBTriggerAttributeBindingProvider.cs

```diff
@@ -185,31 +182,10 @@ private void ThrowMissingConnectionStringException(bool isLeaseConnectionString
                 $"{nameof(CosmosDBTriggerAttribute)}.{nameof(CosmosDBTriggerAttribute.LeaseConnection)}" :
                 $"{nameof(CosmosDBTriggerAttribute)}.{nameof(CosmosDBTriggerAttribute.Connection)}";
 
-            string optionsProperty = $"{nameof(CosmosDBOptions)}.{nameof(CosmosDBOptions.ConnectionString)}";
-
             string leaseString = isLeaseConnectionString ? "lease " : string.Empty;
 
             throw new InvalidOperationException(
-                $"The CosmosDBTrigger {leaseString}connection string must be set either via a '{Constants.DefaultConnectionStringName}' configuration connection string, via the {attributeProperty} property or via {optionsProperty}.");
-        }
-
-        internal string ResolveConnectionString(string unresolvedConnectionString, string propertyName)
-        {
-            // First, resolve the string.
-            if (!string.IsNullOrEmpty(unresolvedConnectionString))
-            {
-                string resolvedString = _configuration.GetConnectionStringOrSetting(unresolvedConnectionString);
-
-                if (string.IsNullOrEmpty(resolvedString))
-                {
-                    throw new InvalidOperationException($"Unable to resolve app setting for property '{nameof(CosmosDBTriggerAttribute)}.{propertyName}'. Make sure the app setting exists and has a valid value.");
-                }
-
-                return resolvedString;
-            }
-
-            // If that didn't exist, fall back to options.
-            return _options.ConnectionString;
+                $"The CosmosDBTrigger {leaseString}connection must be set either via a '{Constants.DefaultConnectionStringName}' configuration or via the {attributeProperty} property.");
```

Comment:
We've changed a similar message in CosmosDBExtensionConfigProvider.cs to "The {attributeProperty} property cannot be an empty value."

Reply:
It's a bit of a different scenario, the `CosmosDBExtensionConfigProvider.cs` validates that a Binding does not have a Connection that is empty (it is set to an empty value). If it's not set, it will attempt later on to use the default config section, and if it's not found, error out.

For Triggers, that Validation step earlier does not occur, so we need here to cover 2 things, that they are not setting it to empty and that the default is present. Hence the message covers both cases.